graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/FederationDirectives.java

```diff
@@ -28,136 +29,89 @@
 public final class FederationDirectives {
     /* Directive locations */
 
+    public static final String keyName = "key";
+    public static final String externalName = "external";
+    public static final GraphQLDirective external = newDirective()
+            .name(externalName)
+            .validLocations(FIELD_DEFINITION)
+            .build();
+
+    /* fields: _FieldSet */
+    public static final String requiresName = "requires";
+    public static final String providesName = "provides";
+    public static final String extendsName = "extends";
+
+    /* directive @key(fields: _FieldSet!) on OBJECT | INTERFACE */
+    public static final GraphQLDirective extends_ = newDirective()
+            .name(extendsName)
+            .validLocations(OBJECT)
+            .build();
+    public static final Set<GraphQLDirective> allDirectives = new HashSet<>();
+    public static final Set<SDLDefinition> allDefinitions = new HashSet<>();
     private static final DirectiveLocation DL_OBJECT = newDirectiveLocation()
             .name("OBJECT")
             .build();
 
+    /* directive @external on FIELD_DEFINITION */
+    public static final DirectiveDefinition extendsDefinition = newDirectiveDefinition()
+            .name(extendsName)
+            .directiveLocations(Arrays.asList(DL_OBJECT))
+            .build();
     private static final DirectiveLocation DL_INTERFACE = newDirectiveLocation()
             .name("INTERFACE")
             .build();
-
     private static final DirectiveLocation DL_FIELD_DEFINITION = newDirectiveLocation()
             .name("FIELD_DEFINITION")
             .build();
 
-    /* fields: _FieldSet */
-
-    private static final GraphQLArgument fieldsArgument = newArgument()
-            .name("fields")
-            .type(new GraphQLNonNull(_FieldSet.type))
+    /* directive @requires(fields: _FieldSet!) on FIELD_DEFINITION */
```

Comment:
Similarly, this comment seems mismatched.

Reply:
Oops, code got "reordered" :(